### DISCUSSION TITLE
Websev Architecture v.1.0

### DISCUSSION THREAD

**Discussion by kimminkyeu** [42seoul-webserv/webserv#28] [Announcements]
@JuneParkCode  @csstring 
피그마 이메일 주소 알려주시면 수정할 수 있게 권한 드리겠습니다.
 https://www.figma.com/file/JoXZuDOmT6HOlwUUFcUO38/Webserv-architecture?node-id=0%3A1&t=f1X0ouGVyNkNNB31-1
<img width="1178" alt="image" src="https://user-images.githubusercontent.com/60287070/203902959-c61e527b-458b-4ca1-ae25-106fb1333ec6.png">

**Comment by JuneParkCode** [comment]
RequestParser에서 RequestProcessor 쪽으로 Header 가 완성되었을 때 확인 요청을 보냅니다.
- Header 가 invalid -> 잘못된 요청이라고 response 바로 함
- valid -> 요청의 body가 필요한 경우 request parser에 데이터를 요청하고, 필요없는 경우 해당 method에 해당하는 작업을 수행함 (기존 로직)

이 부분 도식화가 되었으면 좋겠습니다.

**Reply by kimminkyeu** [reply]
@JuneParkCode 이렇게 수정 하면 될까용

**Reply by JuneParkCode** [reply]
굿굿
